src/components/TabList.js

```diff
@@ -13,7 +22,7 @@ module.exports = React.createClass({
   },
 
   render() {
-    const { className, children, ...attributes } = this.props;
+    const { className, ...attributes } = this.props;
```

Comment:
This will lead to that attributes, contains `activeTabClassName, disabledTabClassName and children`.
I think the best would be if you do this ti circumvent:
```js
const { className, activeTabClassName, disabledTabClassName, children, ...attributes } = this.props;

...

{renderChildren({ activeTabClassName, disabledTabClassName, children })}